CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## [1.7.2] - 2023-08-16
+
+### Added
+
+- `aws_s3` BundlePublisher plugin (#4355)
+- Telemetry in experimental Agent LRU cache (#4335)
+- Telemetry in Agent Delegated Identity API (#4399)
+- Documentation improvements (#4336, #4407)
+
+### Fixed
+
+- Server no longer unnecessarily activates its CA a second time on startup (#4368)
+- SPIRE Server bundle endpoint now correctly includes bundle sequence number (#4389)
```

Comment:
I'd also be ok with this under "Added". The lack of sequence number was an intentional choice, not something that broke. 🤷